parser/src/test/resources/schemas/TestInvalidTypeRecord2.avsc

```diff
@@ -0,0 +1,11 @@
+{
+  "type": "notRecord",
```

Comment:
umm.. you already have an invalid record type above? is this necessary?

Reply:
this one has a type that is at least a string, just not "record". the one above has type 42 (an int). those are different error paths to test. will add a doc

Reply:
ok